appveyor.yml

```diff
@@ -15,6 +15,15 @@ shallow_clone: true
 test: off
 
 install:
+  # Work around age/brokenness of Appveyor's msys2 by adding the new
+  # maintainer's keyring.
+  - bash -lc "curl -O http://repo.msys2.org/msys/x86_64/msys2-keyring-r21.b39fb11-1-any.pkg.tar.xz"
+  - bash -lc "curl -O http://repo.msys2.org/msys/x86_64/msys2-keyring-r21.b39fb11-1-any.pkg.tar.xz.sig"
+  - bash -lc "pacman-key --verify msys2-keyring-r21.b39fb11-1-any.pkg.tar.xz.sig"
+  - bash -lc "pacman -U --noconfirm msys2-keyring-r21.b39fb11-1-any.pkg.tar.xz"
+  - bash -lc "pacman -Syu --noconfirm"
+  - taskkill /f /fi "MODULES eq msys-2.0.dll"
+  - bash -lc "pacman -Syu --noconfirm"
```

Comment:
I think these -Syus aren't necessary, since we do a -Syyuu and -Suu further down. The taskkill probably is though. It should be between our -Syyuu and -Suu.